app/views/shared/_top_banner.html.erb

```diff
@@ -3,6 +3,15 @@
 </div>
 
 <div class="navigationBar">
-    <%= link_to "Login", "/login" %>
-    <%= link_to "Logout", "/logout", method: :post %>
-</div>
\ No newline at end of file
+	<%= link_to "Home", @user.admin? ? "/dashboard/admin" : "/dashboard/user" %>
+	<%= link_to "Users", "/login" %>
+	<%= link_to "General Committee", "/login" %>
+	<%= link_to "Login", "/login" %>
+  <%= link_to "Logout", "/logout", method: :post %>
+	<div class="dropdown">
+		<button class="dropbtn">Comittees</button>
+		<div class="dropdown-content">

```

Comment:
Instead of having just "Committee Test", can we loop through Committee.all and make link for each committee?